desktop/lib/store-kit/index.js

```diff
@@ -0,0 +1,63 @@
+var $ = require( 'nodobjc' );
+
+$.framework( 'Foundation' );
+$.framework( 'StoreKit' );
+
+function log() {
+	console.log.apply( null, [ 'StoreKit:' ].concat( [].slice.call( arguments ) ) );
+}
+
+function requestProducts( onReceiveResponse ) {
+	var pool = $.NSAutoreleasePool( 'alloc' )( 'init' );
+
+	var ProductsRequestDelegate = $.NSObject.extend( 'ProductsRequestDelegate' );
+
+	ProductsRequestDelegate.addMethod( 'requestDidFinish:', 'v@:@', function( self, cmd, request ) {
+		log( 'Executing:', cmd );
+	} );
+
+	ProductsRequestDelegate.addMethod( 'productsRequest:didReceiveResponse:', '@@:@@', function( self, cmd, request, response ) {
+		var products = response( 'products' ),
+			invalidProductIds = response( 'invalidProductIdentifiers' ),
+			productsCount = products( 'count' ),
+			firstProductTitle = 'none';
+
+		log( 'Executing:', cmd );
```

Comment:
If you enable debugging in the app (app menu > enable debug) then you will see `debug` calls on the terminal. You can then use `debug` instead of your own log function.